src/components/elements/ombed.tsx

```diff
@@ -5,6 +5,7 @@ import Embed from "react-tiny-oembed"
 import {HtmlHTMLAttributes} from "react"
 import {useIntersectionObserver} from "usehooks-ts"
 import {twMerge} from "tailwind-merge"
+import "@styles/form.css"
```

Comment:
The styles does get added to the page, but it's not really applicable for the oembed component.

It might just need to get added to the WYSWYG component.

Reply:
It actually isn't even necessary at all. Somehow the style sheet is getting loaded automatically

Reply:
Looks like the tsconfig.json addition of `@styles/` is adding the stylesheet all the time

Reply:
Ooo @pookmish  is that what we want though? 🤔 Should I remove the import or should I switched back to the direct import and move it to the WYSIWYG instead?